internal/model/workload.go

```diff
@@ -5,16 +5,23 @@ import (
 
 	"github.com/lib/pq"
 	"github.com/redhatinsights/ros-ocp-backend/internal/types/workload"
+	"gorm.io/gorm"
 )
 
 type Workload struct {
 	ID              uint `gorm:"primaryKey;not null;autoIncrement"`
-	ClusterID       uint
-	Cluster         Cluster
+	ClusterID       string
+	Cluster         Cluster               `gorm:"foreignKey:ClusterID" json:"-"`
```

Comment:
@patilsuraj767 I have added explicit fk relations in `workload` and `recommendation_set`, to ensure we don't
face any ambiguity with `Preload` db queries.
Hope, that is okay.